src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -177,6 +226,12 @@ trait EntityComponent {
       unmarshalEntities(sql)
     }
 
+    def list(workspaceContext: SlickWorkspaceContext, entityRefs: Traversable[AttributeEntityReference]): ReadAction[TraversableOnce[Entity]] = {
+      val baseSelect = sql"""#$baseEntityAndAttributeSql where e.#${quoteIdentifier("workspace_id")} = ${workspaceContext.workspaceId} and (e.#${quoteIdentifier("entity_type")}, e.#${quoteIdentifier("name")}) in ("""
+      val entityTypeNameTuples = entityRefs.map { case ref => sql"(${ref.entityType}, ${ref.entityName})" }.reduce((a, b) => concatSqlActionsWithDelim(a, b, sql","))
+      unmarshalEntities(concatSqlActions(concatSqlActions(baseSelect, entityTypeNameTuples), sql")").as[EntityListResult])
```

Comment:
suggestion:
```scala
val actions = concatSqlActions(concatSqlActions(baseSelect, sql"("), concatSqlActions(entityTypeNameTuples, sql")")
```
do decrease the distance between the opening and closing paren for clarity. Also, concatSqlActions should probably take 2 or more arguments.